client/results.html

```diff
@@ -9,12 +9,15 @@
     <title>Not Google</title>
 </head>
 <body>
-    <h1>
+    <h1 class ="google resultsHeader">
         <span style="color:#DB4437">N</span><span style="color:#4285F4">o</span><span style="color:#F4B400">t</span>
```

Comment:
h1 tag much longer than text., anywhere i click on the top re-routes me back to homepage